app/src/main/java/com/T05/krowdtrialz/ui/experimentDetails/ExperimentStatistics.java

```diff
@@ -74,16 +80,100 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
      *  Ricky Au
      */
     private void populateTrialResults(View view) {
+        String type = experiment.getType();
+        String stdString = "NaN";
+        String meanString = "NaN";
+        String q0String = "NaN";
+        String q1String = "NaN";
+        String q2String = "NaN";
+        String q3String = "NaN";
+        String q4String = "NaN";
+
+        Double stdDouble = 0.0;
+        Double meanDouble = 0.0;
+        Double q0Double = 0.0;
+        Double q1Double = 0.0;
+        Double q2Double = 0.0;
+        Double q3Double = 0.0;
+        Double q4Double = 0.0;
+
+        if (type.equals("Measurement")) {

```

Comment:
Could you change this to be `type.equals(MeasurementExperiment.type)` to stay away from literals